x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/client/SecurityClient.java

```diff
@@ -358,6 +361,10 @@ public void invalidateApiKey(InvalidateApiKeyRequest request, ActionListener<Inv
         client.execute(InvalidateApiKeyAction.INSTANCE, request, listener);
     }
 
+    public void getApiKey(GetApiKeyRequest request, ActionListener<GetApiKeyResponse> listener) {
```

Comment:
So we typically also have a prepareGetApiKey method that returns a GetApiKeyRequestBuilder. I think you have done the right thing with the request and it being final. Also given the depreciation of the transport client, I am ok with not having this